addon/-private/system/store/finders.js

```diff
@@ -55,7 +55,8 @@ export function _find(adapter, store, typeClass, id, internalModel, options) {
 
 export function _findMany(adapter, store, typeClass, ids, internalModels) {
   let snapshots = Ember.A(internalModels).invoke('createSnapshot');
-  let promise = adapter.findMany(store, typeClass, ids, snapshots);
+  let unique_ids = Ember.A(ids).uniq();
```

Comment:
Should this be `uniqueIds`? Variable names are typically camelCased in Javascript.